stripe/src/main/java/com/stripe/android/model/Source.java

```diff
@@ -117,6 +117,7 @@
     private SourceReceiver mReceiver;
     private SourceRedirect mRedirect;
     private @SourceStatus String mStatus;
+    private Map<String, Object> mSourceTypeData;
```

Comment:
Yeah, figured I'd do it if needed in a follow up. With this diff, I have all the tools I need to ping api/v1/sources with parameters, and get a source token back. (No acting on redirects yet)